lib/bambora/www_form_client.rb

```diff
@@ -13,20 +13,25 @@ class WWWFormClient < Bambora::RestClient
     # @param api_key [String] Indicating the API Key to be used with the request.
     #
     # @return [Hash] Indicating success or failure of the operation.
-    def post(path:, body:, api_key:)
+    def post(path:, body:)
+      # Both Faraday's and Excon's docs show that you can pass a hash into the +body+ and set the content type to
+      # application/x-www-form-urlencoded and the +body+ will be transformed into query parameters, however, this
+      # did not work in testing so I am manually transforming the hash into query parameters here.
       parse_response_body(
         super(
           path: "#{path}?#{WWWFormParameters.new(body: body)}",
           body: nil,
-          headers: build_headers(api_key: api_key),
+          headers: build_headers,
```

Comment:
Why aren't we using the `api_key` anymore?

Reply:
The API key gets merged into the `body` for this request before we pass them into the client here.

Reply:
This endpoint is so bizarre. I guess that's legacy code for you 🤷‍♂.